copy_database.yml

```diff
@@ -82,7 +82,7 @@
       environment:
         DB_URL: "postgresql://{{ archive_db_user }}:{{ archive_db_password }}@{{ archive_db_host }}:{{ archive_db_port }}/{{ db_name }}"
         DB_SUPER_URL: "postgresql://postgres:{{ archive_db_password }}@{{ archive_db_host }}:{{ archive_db_port }}/{{ db_name }}"
-      command: "/var/cnx/venvs/publishing/bin/cnx-db venv"
+      command: "/var/cnx/venvs/archive/bin/cnx-db venv"
```

Comment:
Hm... is this meant to run locally on the database server?  Do we need to add `delegate_to: "{{ groups.database|first }}"`?

Reply:
The underlying code will refuse to install the path-to-venv if it's not a local DB connection, ISTR.

Reply:
BTW, this came about when Scott ran the db copy to staging. Production _has_ the proper path to the archive venv. Not sure how a path to a non-existent venv got installed.

Reply:
Then we need the `delegate_to` line